web/help/GuideUnit1.md

```diff
@@ -1896,12 +1894,37 @@ y axis, since those are the angles in between the numbers -90 and 90.  But the
 expression `arc(270, 90, 5)` covers the angles to the left of the y axis,
 because those are the numbers between 270 and 90.
 
+To see a half circle with radius 5 drawn on the corrdinate plane, use this code:
+
+~~~~~ . clickable
+program   = drawingOf(halfCircle & coordinatePlane)
+halfCircle=arc(-90,90,5)
+~~~~~
+
 Following the same pattern as most other shapes, the `thickArc` function is
 just like `arc`, except that you give it one extra number representing the
-thickness of the line to draw.  The `sector` function is sort of like the
+thickness of the line to draw.
+
+To draw half of a tire, use this code:
+
+~~~~~ . clickable
+program   = drawingOf(halfTire & coordinatePlane)
+halfTire=thickArc(-90,90,5,4)
+~~~~~
+
+Notice the thickness is split between both sides of the arc.
```

Comment:
I'm not sure it will be clear what you mean by this, and you may need to be more explicit and refer to the coordinate plane: for example, by pointing out that when the curve intersects the x axis, it is centered around an x value of 5, but it extends all the way from 3 through 7.  It could be useful to copy the image with the coordinate plane, and show it here.

(Copying an image is a little tricky.  Basically, you want to run the program, right-click on it, then wait for the on-screen controls to go away before choosing "Save image as..." in your browser.  Then you can add the file to your git branch in the `web/help` directory, and then refer to it from Markdown.  Alternatively, you can use the iframe pattern to just run the program inline.  Either will work, though I normally try to save the image unless I am going for some kind of animation or interactive demo.)

Reply:
Yeh. I was having trouble with the wording but though it was important to point out how the thickness worked. I'll add more of a description and an image.